examples/quantizing_moe/deepseek_recipe_w4a16.yaml

```diff
@@ -0,0 +1,9 @@
+quant_stage:
+  quant_modifiers:
+    GPTQModifier:
+      sequential_update: true
+      ignore: [lm_head, "re:.*mlp.gate$"]
+      config_groups:
+        group_0:
+          weights: {num_bits: 4, type: int, symmetric: true, strategy: channel, dynamic: false}
```

Comment:
We may want to use grouped and actorder for the final example based on how the accuracy comes out, but this is good for now